ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -344,3 +357,25 @@ def add_disk_for_vsphere_platform():
             raise NotImplementedError(
                 "LSO Deployment for VMDirectPath is not implemented"
             )
+
+
+def add_disk_for_rhv_platform():
+    """
+    Add disk for RHV platform
+
+    """
+    platform = config.ENV_DATA.get("platform").lower()
+    if platform == constants.RHV_PLATFORM:
+        # Importing here to avoid circular dependency
+        from ocs_ci.deployment.rhv import RHVBASE
+
+        rhv_base = RHVBASE()
+        rhv_base.attach_disks(
+            config.ENV_DATA.get("device_size", defaults.DEVICE_SIZE),
+            config.ENV_DATA.get("disk_format", constants.RHV_DISK_FORMAT_RAW),
+            config.ENV_DATA.get(
+                "disk_interface", constants.RHV_DISK_INTERFACE_VIRTIO_SCSI
+            ),
+            config.ENV_DATA.get("sparse", False),
+            config.ENV_DATA.get("pass_discard", True),
```

Comment:
we need to check what is the default value in the SDK for `pass_discard`. This could impact the performance.

Reply:
As per offline discussion, these default values were specifically chosen in dev environment for ISCSI domain.
Therefore I will make the changes to use default value here as we are not using that kind of storage.
We will still be able to pass any value later via a specific conf file if required.